packages/perseus/src/util/graphie.ts

```diff
@@ -1771,7 +1771,14 @@ const GraphUtils = {
     },
 
     // Find the angle in degrees between two or three points
-    findAngle: function (point1: Coord, point2: Coord, vertex?: Coord) {
+    // This function is deprecated as it has several issues calculating
+    // correctly when dealing with reflexive angles or the position of point2
+    // (LEMS-2202) Remove this function while removing the Legacy Interactive Graph.
+    findAngleDeprecated: function (
```

Comment:
I can add a comment to LEMS-2202 to mention deleting this function once we strip out the Legacy Graph. First, I need to confirm if `interactive.ts` is used anywhere else as it's the only other consumer of this function.

Reply:
Praise: Thanks for making it clear this function should not be used in the future!